pom.xml

```diff
@@ -53,6 +53,11 @@
         <maven.build.timestamp.format>E MMM dd hh:mm:ss zzz yyyy</maven.build.timestamp.format>
         <build.timestamp>${maven.build.timestamp}</build.timestamp>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <sonar.projectKey>Datical_liquibase</sonar.projectKey>
```

Comment:
@jnewton03 Is projectName optional?I noticed it is not there in other repos like storedlogic and AppDBA